requirements.txt

```diff
@@ -1,6 +1,6 @@
 # Requirements needed to run the app.
-flask
-Flask-WTF
-flask-restful
+Flask==1.1.2
+Flask-RESTful==0.3.8
+Flask-WTF==0.14.3
+func-adl==1.0.0a18
 func-adl-uproot==0.11
```

Comment:
@masonproffitt or @gordonwatts - does it make sense to reference both func-adl and func-adl-uproot independently here, or is func-adl a dependency of func-adl-uproot and should come from there?

Reply:
func_adl_uproot currently has no dependency on func_adl